Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -896,13 +896,14 @@ final class VisibleItemsProvider {
     currentMonth = month
     currentMonthFrame = monthFrame
     while
-      bounds.contains(CGPoint(x: currentMonthFrame.maxX, y: currentMonthFrame.maxY)),
+      bounds.contains(
+        CGPoint(x: currentMonthFrame.maxX - 1, y: currentMonthFrame.maxY - 1)
```

Comment:
`CGRect`'s `containsPoint` function won't return `true` if the point lies on the trailing/bottom edge